tutorials/algorithms/03_vqe_simulation_with_noise.ipynb

```diff
@@ -226,7 +226,7 @@
     "from qiskit.test.mock import FakeVigo\n",
     "device_backend = FakeVigo()\n",
     "\n",
-    "backend = Aer.get_backend('qasm_simulator')\n",
+    "backend = Aer.get_backend('aer_simulator')\n",
     "counts1 = []\n",
     "values1 = []\n",
     "noise_model = None\n",
```

Comment:
Is disabling parallelism still necessary? If yes, do we want to add a comment? 🙂

Reply:
Removed statement